graylog2-web-interface/src/pages/EditContentPackPage.jsx

```diff
@@ -35,11 +35,16 @@ const EditContentPackPage = createReactClass({
   },
 
   componentDidMount() {
-    ContentPacksActions.getRev(this.props.params.contentPackId, this.props.params.contentPackRev).then(() => {
+    ContentPacksActions.get(this.props.params.contentPackId).then(() => {
+      const originContentPackRev = this.props.params.contentPackRev;
+      const newContentPack = this.state.contentPack[originContentPackRev];
+      const nextContentPackRev = Math.max(...Object.keys(this.state.contentPack).map(x => parseInt(x, 10))) + 1;
```

Comment:
Bumping the revision in the UI can go wrong when two people clone the same content pack. But since we have unique indices in place in the database, the worst thing that can happen is that the user gets an ugly error. :wink: